### PR TITLE
Issue #7492: enforce jdk 1.8 by maven enforcer plugin

### PR DESCRIPTION
**Pull request by romani**
Issue #7492

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1069,23 +1069,23 @@
...
+                  <maxJdkVersion>1.8</maxJdkVersion>
                 </enforceBytecodeVersion>
                 -->
               </rules>
             </configuration>
           </execution>
         </executions>
-        <!-- we can not use this as it require
-        compilation on 1.6 only, due to dependency to 'tool'
+        <!-- required dependency for enforceBytecodeVersion  -->
```
Comment (rnveach) [inline pom.xml]: Why is it commented out if it is required?

Comment (romani) [reply pom.xml]: because enforceBytecodeVersion is commented out

Comment (rnveach) [reply pom.xml]: then we should keep the comment that says we can't use. that was removed and all thats left is "this commented out section is required".

Comment (romani) [reply pom.xml]: please recheck new wording.

**Comment by rnveach** [comment]
@romani CI is failing.

**Pull request merged**